TenPrintCover/C64Converter.m

```diff
@@ -0,0 +1,49 @@
+//
+//  C64Converter.m
+//  TenPrintCover
+//
+//  Created by Raman Singh on 2021-03-11.
+//  Copyright © 2021 NYPL_Labs. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "C64Converter.h"
+
+@implementation C64Converter
+
++ (NSString *)convertToC64:(NSString *)value {
```

Comment:
nit: this could be just called `convert` since the "to64" part is implied from the class name.

Reply:
Agreed. Done.